project/agent-groups/components/AgentGroupForm.vue

```diff
@@ -0,0 +1,244 @@
+<template>
+  <v-form
+    ref="form"
+    v-model="isFormValid"
+    @submit.prevent="onSubmit"
+  >
+    <div style="padding-bottom: 20px;">
```

Comment:
Use class `pb-X` instead of `style="padding-bottom: 20px;`

https://vuetifyjs.com/en/styles/spacing/#how-it-works